core/src/main/scala/org/apache/spark/shuffle/IndexShuffleBlockManager.scala

```diff
@@ -106,7 +106,7 @@ class IndexShuffleBlockManager(conf: SparkConf) extends ShuffleBlockManager {
     // find out the consolidated file, then the offset within that from our index
     val indexFile = getIndexFile(blockId.shuffleId, blockId.mapId)
 
-    val in = new DataInputStream(new FileInputStream(indexFile))
+    val in = new DataInputStream(new BufferedInputStream(new FileInputStream(indexFile)))
```

Comment:
Similar to other comments made by @srowen and @andrewor14, here we're only reading 2 longs.  It doesn't seem like the buffering is necessary?